src/base/types.h

```diff
@@ -42,5 +42,6 @@ enum class ShutdownAction
 };
 
 typedef QMap<QString, QString> QStringMap;
+using QByteArrayMap = QMap<QByteArray, QByteArray>;
```

Comment:
>This is the same in c++11:

OK. But from the my view point, it comes down to whether you are going to define a type, which is also stated in what you quote:

>it does not define a new type and ...

In this specific scenario, I interpret the author is defining a type, because `QByteArrayMap` is used non locally.
And another weaker argument, the `typedef` is just above it... mixing like this is weird IMO.